public_src/store/model.ts

```diff
@@ -16,4 +16,5 @@ export interface IAppState {
   errorCorrectionMode: ISuggestionsBrowserOptions;
   beginnerView: string;
   switchMode: boolean;
+  wizardMode: string;
```

Comment:
you send also `this.appActions.actSetWizardMode(null);` which is not string, so `string | null | 'router wizard' | 'router master wizard'` to make it less confusing?